src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/synthetic-monitoring-best-practices-guide.mdx

```diff
@@ -18,34 +18,38 @@ New Relic's synthetic monitoring—powered by a Selenium-driven Chrome browser
 
 Synthetic monitors are virtual browsers that measure the performance of your website, recording each check in detail. They also capture aggregate numbers for load time, uptime, and average download size, as well as an overview, detailed statistics for each page resource, and downtime incidents.
 
-There are four types of synthetic monitors; the ones you deploy will depend on the things you want to monitor:
-
-* **Ping monitors**—to ensure that your site is accessible.
-* **Simple browser monitors**—to ensure end-user performance.
-* **Scripted browsers**—to ensure that particular resources are present.
-* **API monitors**—to ensure that your app server works as well as your website.
+There are [seven types of synthetic monitors](/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors). Which monitors you choose to deploy will depend on what you want to monitor and how you want to create the monitor. For example, a Step monitor is a great way to create what is essentially a Scripted browser monitor behind the curtains, without having to write code.
```

Comment:
```suggestion
We have [seven types of synthetic monitors](/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors) to choose from depending on what you want to monitor and how you'd like to do it. For example, a [step monitor](/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors#stepmonitor) is a great way to create what essentially functions as a scripted browser without having to write code. If you're looking for programmatic management of your monitors, you can also bulk create, edit, or delete your synthetic monitors with our [NerdGraph API](/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial). 
```

Reply:
I like that you deleted the list. Added our NerdGraph option as a counter-option to step monitors and made some changes to style.